packages/client/ui/react-native/README.md

```diff
@@ -1,67 +1,155 @@
   return (
-    <CrossmintProvider
-      apiKey="YOUR_API_KEY"
-    >
+    <CrossmintProvider apiKey={process.env.EXPO_PUBLIC_CROSSMINT_API_KEY}>
       <CrossmintAuthProvider>
-        {/* Your app content */}
+        <CrossmintWalletProvider
+          createOnLogin={{
+            chain: "solana",
+          }}
+        >
+          <MainApp />
+        </CrossmintWalletProvider>
       </CrossmintAuthProvider>
     </CrossmintProvider>
   );
 }
 ```
 
-### Authentication Hooks
+**Option B: 🔧 Bring Your Own Authentication**
 
-Once the providers are set up, you can use the authentication hooks in your components:
+Already have authentication? Skip Crossmint Auth and use wallets with your existing system:
 
 ```tsx
-import { useCrossmintAuth } from "@crossmint/client-sdk-react-native-ui";
+import {
+  CrossmintProvider,
+  CrossmintWalletProvider
+} from "@crossmint/client-sdk-react-native-ui";
+
+export default function App() {
+  return (
+    <CrossmintProvider apiKey={process.env.EXPO_PUBLIC_CROSSMINT_API_KEY}>
+      {/* No CrossmintAuthProvider needed! */}
+      <CrossmintWalletProvider>
+        <MainApp />
+      </CrossmintWalletProvider>
+    </CrossmintProvider>
+  );
+}
+```
 
-function ProfileScreen() {
-  const { user, status, logout, login } = useCrossmintAuth();
+### 2. Use Authentication & Wallets
 
-  if (status === "logged-out") {
+The React Native SDK uses [Expo's SecureStore](https://docs.expo.dev/versions/latest/sdk/securestore/) for secure, encrypted storage of authentication tokens. This provides a platform-native secure storage solution that encrypts sensitive data on the device.
+
+```tsx
+import { View, Button, Text } from "react-native";
+import { useCrossmintAuth, useWallet } from "@crossmint/client-sdk-react-native-ui";
+
+export default function MainApp() {
+  const { loginWithOAuth, logout, user } = useCrossmintAuth();
+  const { wallet, status } = useWallet();
+
+  if (!user) {
     return (
-      <View>
-        <Button
-          title="Sign in with Google"
-          onPress={() => login("google")}
+      <View style={{ padding: 20 }}>
+        <Button 
+          title="Login with Google" 
+          onPress={() => loginWithOAuth("google")} 
         />
       </View>
     );
   }
 
-  return (
-    <View>
-      <Text>Welcome, {user?.email}</Text>
-      <Button title="Logout" onPress={logout} />
-    </View>
-  );
+  if (status === "loaded") {
+    return (
+      <View style={{ padding: 20 }}>
+        <Text>Welcome {user.email}!</Text>
+        <Text>Wallet: {wallet?.address}</Text>
+        <Button 
+          title="Send 1 USDC" 
+          onPress={() => wallet?.send(recipient, "usdc", "1.0")} 
+        />
+        <Button title="Logout" onPress={logout} />
+      </View>
+    );
+  }
+
+  return <Text>Loading wallet...</Text>;
 }
 ```
 
+## 🔧 Bring Your Own Authentication
+
+**Simple Setup:** Remove `CrossmintAuthProvider` and create wallets directly for your users.
+
+### Why Use Your Own Auth?
```

Comment:
I think we can remove this. You already have the example above and there's too many nice checks for BYOA